examples/kubernetes.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: red
+  namespace: default
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: red
+    spec:
+      containers:
+      - name: red
+        image: segment/red:v0.2.0
+        args: [
+            "proxy",
+            "-upstream", "consul://consul.default.svc.cluster.local:8500/redis?cluster=dev",
```

Comment:
Is this the address of the consul server? For the demo it's probably fine but we should have consul agents on the hosts and point the proxy at those instead of the main consul servers.